### PR TITLE
Improvement: add Zarc element

### PR DESCRIPTION
**Pull request by etrevis**
RQ element defined with resistance and relaxation time as parameters. This element should cover well:

_insert a boundary in the frequency of RC and RQ elements #211_

### CODE REVIEW

File: impedance/models/circuits/elements.py [hunk 1]
```diff
@@ -313,6 +313,25 @@ def K(p, f):
     return Z
 
 
+@element(num_params=3, units=['Ohm', 'sec', ''])
+def Zarc(p, f):
+    """ An RQ element rewritted with resistance and
```
Comment (mdmurbach) [inline impedance/models/circuits/elements.py:318]:
```suggestion
    """ An RQ element rewritten with resistance and
```

**Review by mdmurbach — APPROVED** [review]
Looks great! Huge thanks for adding this in :)

**Comment by mdmurbach** [comment]
@all-contributors add @etrevis for code

**Comment by nicmazza** [comment]
I only saw this now, and it does seem to do what I was lookign for! thanks

**Pull request merged**